stringology/README.md

```diff
@@ -8,3 +8,6 @@
       also a survey paper that is more than the sum of its citations,
       clarifying the presentation of all the algorithms with a
       unifying framework.
+
+* [Fast String Searching](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.13.9460&rep=rep1&type=pdf)
```

Comment:
i was wondering why this is 404ing, but then i saw that citeseerx.ist.psu.edu as a whole is

> WE ARE SORRY BUT CITESEERX IS UNDER MAINTENANCE.